modelplugin/aether-4.0.0/modelplugin/modelplugin.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2021-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package modelplugin - A plugin for the YGOT model of aether-4.0.0.
+// TODO: Remove this - not needed anymore - check sdcore-adapter
+//  All this metadata is now kept in the sdran-helm-charts/config-models/aether-4.0.0
+package modelplugin
+
+import (
+	"fmt"
+
+	"github.com/onosproject/config-models/modelplugin/aether-4.0.0/aether_4_0_0"
+	"github.com/openconfig/gnmi/proto/gnmi"
+	"github.com/openconfig/goyang/pkg/yang"
+	_ "github.com/openconfig/ygot/genutil"
+	_ "github.com/openconfig/ygot/ygen"
+	"github.com/openconfig/ygot/ygot"
+	_ "github.com/openconfig/ygot/ytypes"
+)
+
+type Modelplugin string
+
+const modeltype = "Aether"
+const modelversion = "4.0.0"
+const modulename = "aether.so.4.0.0"
+
+var ModelData = []*gnmi.ModelData{
+	{Name: "connectivity-service", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+	{Name: "enterprise", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+
+	{Name: "aether-types", Organization: "Open Networking Foundation", Version: "2021-09-10"},
+	{Name: "ap-list", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+	{Name: "application", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+	{Name: "device-group", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+	{Name: "ip-domain", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+	{Name: "site", Organization: "Open Networking Foundation", Version: "2021-09-10"},
+	{Name: "upf", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+	{Name: "vcs", Organization: "Open Networking Foundation", Version: "2021-09-10"},
+	{Name: "template", Organization: "Open Networking Foundation", Version: "2021-09-10"},
```

Comment:
`traffic-class` should be `2021-09-10` and `template` should be `2021-06-02`

Reply:
Template will need it's version updated in roc-helm-charts. I'll set it back to 2021-06-02 and put that on the to-do list.